MARKETING/PressReleaseDrafts/2023/draft.md

```diff
@@ -32,20 +32,14 @@ This allows WoT applications to be written on top of a single, portable interact
 
 <!-- What is new? -->
 ## Significant improvements and functionality
-In general, these new Recommendations have many small changes to improve the usability of
-the previous release and address new use cases. However, significant new features are also defined,
+These new Recommendations were revised to improve the usability of
+the previous release and address new use cases. Significant new features were also defined,
 including Thing Models and Discovery.
 
-Thing Models provide a way to describe classes of things, such as product lines from particular vendors.
-Thing Models also support modularity, allowing a parameterized model to be built from reusable components.
-A Thing Model can be used to generate a Thing Description by providing the appropriate parameters.
+Thing Models provide a way to describe classes of things, such as product lines from particular vendors, and support modularity by allowing a parameterized model to be built from reusable components. A Thing Model can be used to generate a Thing Description by providing the appropriate parameters.
```

Comment:
I was trying to avoid long run-on sentences, but combining these two is ok as they are related concepts.

Reply:
I'm also ok with this change

Reply:
"product lines" is very special... I added "sensor devices or product lines" in the PR #1152